lib/realmq.js

```diff
@@ -21,17 +22,19 @@ class RealMQ {
   }
 
   /**
-   * @param authToken
-   * @param [host]
-   * @param [autoConnect]
-   * @param [autoSubscribe]
+   * @param {string} authToken
+   * @param {string} [host]
+   * @param {boolean} [autoConnect]
+   * @param {boolean} [autoSubscribe]
+   * @param {boolean} [autoSyncSubscriptions]
    */
   constructor(
     authToken,
     {
       host = RealMQ.DEFAULT_HOST,
       autoConnect = false,
       autoSubscribe = false,
+      autoSyncSubscriptions = false,
```

Comment:
In my eyes `autoSync` makes only sense when auto subscribe is active.
I think it autoSync should default to true when autoSubscribe is enabled.